index.js

```diff
@@ -64,6 +64,16 @@ program.command('importDir <directoryPath>')
   })
 })
 
+program.command('generateFiles [path] [filesCount]')
+.description('Generates a given number of small files.')
+.action((path = '/', filesCount = 10) => {
+  const {url, token} = program
+  const ach = new ACH(token, url, ['io.cozy.files'])
+  ach.connect().then(() => {
+    return ach.createFiles(path.toString(), parseInt(filesCount))
```

Comment:
Actually I think it's totally unnecessary, i'll remove it. Since `filesCount ` needed to be casted I casted both, but the variable is a string anyway.